Generator.Equals/Generator.Equals.csproj

```diff
@@ -7,8 +7,9 @@
         <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
         <!-- Do not include the generator as a lib dependency -->
         <IncludeBuildOutput>false</IncludeBuildOutput>
-        <LangVersion>9.0</LangVersion>
+        <LangVersion>latest</LangVersion>
         <Nullable>enable</Nullable>
+<!--        <IsExternalInit>true</IsExternalInit>-->
     </PropertyGroup>
```

Comment:
Missing:
```xml
<IsRoslynComponent>true</IsRoslynComponent>
<AnalysisLevel>latest-all</AnalysisLevel>
<EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
```

Also, add a package reference to `PolySharp` if you want to use `<LangVersion>latest</LangVersion>`